### PR TITLE
Freeland Lesson 02 Assignment

### PR DESCRIPTION
**Pull request by ryanfree1994**
I am submitting a pull request for my lesson 02 assignment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,9 +36,9 @@ Follow these steps to get your initial setup started
 - Clone the fork to your local machine
 
 ### Adding the Upstream to your Fork
-- Navigate to your fork's root folder in terminal (e.g. `cd ~/Sites/uw/html300`)
+- Navigate to your fork's root folder in terminal (e.g. `cd ~/Sites/uw/html300-v2`)
 - Make sure you are on your master branch, `git checkout master`
-- Add the upstream repository as a remote connection `git remote add upstream git@github.com:UWFront-End-Cert/html300.git`
+- Add the upstream repository as a remote connection `git remote add upstream git@github.com:UWFront-End-Cert/html300-v2.git`
```
Comment (cherimarie) [inline README.md:41]: Good catch! Would you mind submitting a pull request back to the UW fork of this repo with just these link changes?

File: lesson02/assignment/css/style.scss [hunk 2]
```diff
@@ -4,3 +4,97 @@
 @import 'settings.responsive';
 @import 'components.content';
 
+.main{
+  background-color: $gray;
+  width: 951px;
+  height: 474px;
```
Comment (cherimarie) [inline lesson02/assignment/css/style.scss]: One of the main pillars of responsive design is flexible sizing units. Almost all sizing should use rem or %.

File: lesson02/assignment/css/style.scss [hunk 3]
```diff
@@ -4,3 +4,97 @@
...
+    }
+  }
+  &__h1{
+    font-size: 26px;
+    color: darken($gray, 40%);
+  }
+  &__p{
+    color: darken($gray, 20%);
+    font-style: italic;
+    font-size: 20px;
```
Comment (cherimarie) [inline lesson02/assignment/css/style.scss]: Setting font size to a static px size will make life really difficult for low-vision users who need to zoom in to read text. When setting font size, always use % or rem (which are effectively the same thing, as "rem" is root font size).

File: lesson02/assignment/css/style.scss (hunk 2 above)
Comment (ryanfree1994) [reply lesson02/assignment/css/style.scss]: Great reminder. Thank you!

File: lesson02/assignment/css/style.scss (hunk 3 above)
Comment (ryanfree1994) [reply lesson02/assignment/css/style.scss]: Thank you!!

File: lesson02/assignment/css/_font.variables.scss [hunk 4]
```diff
@@ -0,0 +1,8 @@
+//font-sizes
+$small: 1.000em;
+$medium: 1.250em;
+$large: 1.625em;
+
+//font-styles
+$italic: italic;
+$center: center;
```
Comment (cherimarie) [inline lesson02/assignment/css/_font.variables.scss:8]: This is an excellent use of variables. For a large site, having font information centralized like this makes sitewide style adjustments quick and easy.

Comment (ryanfree1994) [reply lesson02/assignment/css/_font.variables.scss:8]: Thank you! It looks much nicer in the code, too.

**Pull request merged**